echopype/convert/parse_base.py

```diff
@@ -26,8 +27,8 @@ def _print_status(self):
 class ParseEK(ParseBase):
     """Class for converting data from Simrad echosounders.
     """
-    def __init__(self, file, params):
-        super().__init__(file)
+    def __init__(self, file, storage_options, params):
```

Comment:
This is just a comment about consistency: how about reversing the order of `params` and `storage_options`, for consistency with what's used elsewhere? Then, of course, calls to `ParseEK` will have to be updated as well